### PR TITLE
Updates to documentation

### PR DESCRIPTION
**Pull request by serbinsh**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by serbinsh** [comment]
@bcow this update fixes a bug where the workflow crashes because the documentation wasn't updated, therefore the NAMESPACE wasn't up to date.  This is a fix which is probably needed by others.

File: models/linkages/NAMESPACE [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Generated by roxygen2: do not edit by hand
 
+S3method(split,inputs.LINKAGES)
```
Comment (mdietze) [inline models/linkages/NAMESPACE]: this is wrong

File: models/linkages/man/split.inputs.LINKAGES.Rd [hunk 2]
```diff
@@ -4,7 +4,7 @@
 \alias{split.inputs.LINKAGES}
 \title{split.inputs.LINKAGES}
 \usage{
-split.inputs.LINKAGES(settings, start.time, stop.time)
+\method{split}{inputs.LINKAGES}(settings, start.time, stop.time)
```
Comment (mdietze) [inline models/linkages/man/split.inputs.LINKAGES.Rd]: this is wrong

Comment (serbinsh) [inline models/linkages/man/split.inputs.LINKAGES.Rd]: @mdietze @araiho can you take a look?  All I did was re-run the roxygen/make document commands.....

Same here.  I am not sure what was intended?

**Comment by mdietze** [comment]
We'll fix it later -- Roxygen interpreted a function as a S3 method that wasn't

**Pull request merged**